### PR TITLE
Nice formatting of funding source titles in Server admin

### PR DESCRIPTION
**Pull request by blackcoffeexbt**
+ Update to Server admin to use the human readable funding source name.
+ Update to Server admin to sort funding sources alphabetically

### CODE REVIEW

**Review by dni — APPROVED** [review]

File: lnbits/static/js/components/lnbits-funding-sources.js [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Vue.component('lnbits-funding-sources', {
   mixins: [windowMixin],
   props: ['form-data', 'allowed-funding-sources'],
+  methods: {
+    getFundingSourceLabel(item) {
```
Comment (motorina0) [inline lnbits/static/js/components/lnbits-funding-sources.js:5]: why is this needed?
what was being displayed without it?

Comment (blackcoffeexbt) [reply lnbits/static/js/components/lnbits-funding-sources.js:5]: Without this, the [0] value is being displayed e.g. "CoreLightningWallet" this changes the drop down labels to:

CoreLightningWallet > Core Lightning
CoreLightningRestWallet > Core Lightning Rest
LndWallet > Lightning Network Daemon (LND)

etc

**Review by motorina0 — APPROVED** [review]
